docs/user-docs/latest/content-types-builder/configuring-fields-content-type.md

```diff
@@ -0,0 +1,410 @@
+| Maximum length | Tick to define a maximum number of characters allowed.                        |
+| Minimum length | Tick to define a minimum number of characters allowed.                        |
+
+
+:::
+
+::::
+
+### <img width="28" src="../assets/content-types-builder/field-icon_richtext.png"> Rich Text
+
+The Rich Text field displays an editor with formatting options to manage rich text. This field can be used for long written content.
+
+:::: tabs
+
+::: tab Base settings
+
+| Setting name | Instructions                           |
+|--------------|----------------------------------------|
+| Name         | Write the name of the Rich Text field. |
+
+:::
+
+::: tab Advanced settings
+
+| Setting name   | Instructions                                                                |
+|----------------|-----------------------------------------------------------------------------|
+| Default value  | Write the default value of the Rich Text field.                             |
+| Private field  | Tick to make the field private and prevent it from being found via the API. |
+| Required field | Tick to prevent creating or saving an entry if the field is not filled in.  |
+| Maximum length | Tick to define a maximum number of characters allowed.                      |
+| Minimum length | Tick to define a minimum number of characters allowed.                      |
+
+:::
+
+::::
+
+### <img width="28" src="../assets/content-types-builder/field-icon_number.png"> Number
+
+The Number field displays a field for any kind of number: integer, decimal and float.
+
+:::: tabs
+
+::: tab Base settings
+
+| Setting name  | Instructions                                                    |
+|---------------|-----------------------------------------------------------------|
+| Name          | Write the name of the Number field.                             |
+| Number format | Choose between *integer*, *big integer*, *decimal* and *float*. |
+
+:::
+
+::: tab Advanced settings
+
+| Setting name   | Instructions                                                                |
+|----------------|-----------------------------------------------------------------------------|
+| Default value  | Write the default value of the Number field.                                |
+| Private field  | Tick to make the field private and prevent it from being found via the API. |
+| Required field | Tick to prevent creating or saving an entry if the field is not filled in.  |
+| Unique field   | Tick to prevent another field to be identical to this one.                  |
+| Maximum length | Tick to define a maximum number of characters allowed.                      |
+| Minimum length | Tick to define a minimum number of characters allowed.                      |
+
+:::
+
+::::
+
+### <img width="28" src="../assets/content-types-builder/field-icon_date.png"> Date
+
+The Date field can display a date (year, month, day), time (hour, minute, second) or datetime picker.
```

Comment:
Should we clarify the datetime picker as (year, month, day, hour, minute, and second)?